adafruit_hid/__init__.py

```diff
@@ -26,5 +26,14 @@
 
 TODO(description)
 
-* Author(s): Scott Shawcroft
+* Author(s): Scott Shawcroft, Dan Halbert
 """
+# Import these for convenience so that the user may import the packages classes
+# directly without needing to know the file names.
+# E.g.:
+# from adafruit_hid import Keyboard
+
+from .keyboard import Keyboard
+from .keycode import Keycode
+from .mouse import Mouse
+from .us_keyboard_layout import USKeyboardLayout
```

Comment:
Please remove these. It forces all `adafruit_hid` loads to load all of the submodules.

With these imports:
```
Adafruit CircuitPython 0.9.3-1-g48107220-dirty on 2017-04-07; Adafruit Feather M0 Express with samd21g18
>>> import gc
>>> gc.collect()
>>> gc.mem_free()
15632
>>> from adafruit_hid.keycode import Keycode
>gc.collect()
>>> gc.mem_free()
6464
>>> from adafruit_hid.mouse import Mouse
>>> gc.collect()
>>> gc.mem_free()
6432
```

After removing them:
```
Adafruit CircuitPython 0.9.3-1-g48107220-dirty on 2017-04-07; Adafruit Feather M0 Express with samd21g18
>>> import gc
>>> gc.collect()
>>> gc.mem_free()
15584
>>> from adafruit_hid.mouse import Mouse
>>> gc.collect()
>>> gc.mem_free()
13856
>>> from adafruit_hid.keycode import Keycode
>>> gc.collect()
>>> gc.mem_free()
10368
```